src/quickstart/index.md

```diff
@@ -0,0 +1,31 @@
+# Quick Start
+
+Getting started with Deis Workflow in three easy steps.
```

Comment:
I'd change this to "To get started with Deis Workflow, there are three steps to follow:"